docs/web/diff.md

```diff
@@ -0,0 +1,902 @@
+# Comparing analysis results (Diff)
+
+Compare the outstanding reports in a "before" (_baseline_) and an "after" (_new_ or newline) analysis sets, and display the differences in between them the in the same 
```

Comment:
Please start this section with a more descriptive introduction of the report states:

I would rewrite the starting stentence like this:

In CodeChecker a report can be either "Outstanding", or "Closed". Outstanding reports are the reports which should be fixed: are still detected, are unreviewed or confirmed to be true positive.
We call a reports "Closed", if it cannot be detected anymore (e.g. because it is fixed), or it is considered a false positive or an uninteresting (intentional) finding.

CodeChecker diff mechanism always compares two outstanding set of reports, a "before" (_baseline_) and an "after" (_new_ or newline) analysis sets, and display the differences in between them.

Formally:

A report is outsstanding if all ....